AEPMedia/Sources/MediaCollection/MediaCollectionHitGenerator.swift

```diff
@@ -44,10 +44,9 @@ class MediaCollectionHitGenerator {
         self.mediaContext = context
         self.mediaHitProcessor = hitProcessor
         self.mediaConfig = config
-        self.currentRefTS = refTS
-        self.currentPlaybackState = .Init
-        self.currentPlaybackStateStartRefTs = currentRefTS
-
+        self.refTS = refTS
+        self.previousPlaybackState = .Init
```

Comment:
previousPlaybackState name is confusing. We should keep it the same and rename the variable in processPlayback as newPlaybackState.

Reply:
Makes sense 👍